lib/mineflayer.js

```diff
@@ -97,9 +97,9 @@ module.exports = (bot, { viewDistance = 6, firstPerson = false, port = 3000 }) =
       const dx = Math.abs(botX - Math.floor(pos.x / 16))
       const dz = Math.abs(botZ - Math.floor(pos.z / 16))
       if (dx < viewDistance && dz < viewDistance) {
-        const coords = pos.x + ',' + pos.z
-        if (bot._columns[coords]) {
-          const chunk = bot._columns[coords].toJson()
+        const column = bot.world.getColumnAt(pos)
```

Comment:
Pos is not a vec3 it only has x and z

Reply:
https://github.com/PrismarineJS/prismarine-viewer/blob/4df4546918379bcff6cb39e85ab6f1fd09506055/lib/mineflayer.js#L82

Do you mean it's now always a vec3?

Reply:
Erf no its fine I misread